3-networks/envs/prod/README.md

```diff
@@ -8,27 +8,14 @@ The purpose of this step is to setup private and restricted shared VPCs with def
 1. 1-org executed successfully.
 1. 2-environments/envs/prod executed successfully.
 1. 3-networks/envs/shared executed successfully.
+1. Obtain the value for the access_context_manager_policy_id variable. Can be obtained by running `gcloud access-context-manager policies list --organization YOUR-ORGANIZATION_ID`. It is the value under `name`.
```

Comment:
nit: I would just add `--format="value(name)"` so that it is just gives the required value and can be scripted.

Reply:
Good idea. Thanks